### ISSUE TITLE
Improve Reddit post selection algorithm

### ISSUE THREAD

**Issue by TimvdLippe** [WISVCH/florriebot#125]
Right now, the selection is purely `random`, per the implementation of https://hubot.github.com/docs/scripting/#random This thus leads to the potential selection of sub-par posts. Instead, we should give a higher probability to posts with a lot of points. E.g. the probability of a post to be selected should be `post.weight / sum(post.weight)`.

**Comment by praseodym** [comment]
Or maybe use random over just the top n days feed?

**Comment by TimvdLippe** [comment]
Yes that is possible as well. We probably can take the week? https://api.reddit.com/r/autistentoren/top?t=week seems to work and gives back 7 results (only 7 new posts this week). In that case, we can still do the post-filtering with the randomness factor I think.

We would thus retrieve (at most) 25 top posts of a subreddit and then randomly select based on the post score.

**Comment by praseodym** [comment]
Sounds good!
